packages/mgt-chat/src/components/ChatListItem/ChatListItem.tsx

```diff
@@ -155,14 +171,30 @@ export const ChatListItem = ({ chat, myId, onSelected }: IMgtChatListItemProps &
     return String(timestamp);
   };
 
-  const getDefaultProfileImage = () => {
+  const getDefaultProfileImage = (chatObj: Chat) => {
     // define the JSX for FluentUI Icons + Styling
     const oneOnOneProfilePicture = <ChatListItemIcon chatType="oneOnOne" />;
     const GroupProfilePicture = <ChatListItemIcon chatType="group" />;
 
+    const other = chatObj.members?.find(m => (m as AadUserConversationMember).userId !== myId);
+    const otherAad = other as AadUserConversationMember;
```

Comment:
We probably want to have a bot icon for bots which are applications, not conversation members. Or at a minimum they should use the default icon and not the user's icon.

Because the chat component comes from ACS, it doesn't actually send info about the bot, so that control just uses the default icon.